README.md

```diff
@@ -161,14 +161,17 @@ the HTTP functions.
 [lavalink-rs][project:lavalink-rs] or [Songbird][project:songbird] are recommended voice plugins.
 - **default_native_tls**: Default features but using `native_tls_backend`
 instead of `rustls_backend`.
-- **absolute_ratelimits**: Whether the library should use your system clock to avoid
-ratelimits, or use the interval given by Discord that might be less efficient
-due to latency in the network. If you turn this feature on, it is recommended to
-synchronise your clock with an NTP server (such as Google's).
 - **tokio_task_builder**: Enables tokio's `tracing` feature and uses `tokio::task::Builder` to spawn tasks with names if `RUSTFLAGS="--cfg tokio_unstable"` is set.
 - **unstable_discord_api**: Enables features of the Discord API that do not have a stable interface. The features might not have official documentation or are subject to change.
 - **simd_json**: Enables SIMD accelerated JSON parsing and rendering for API calls, use with `RUSTFLAGS="-C target-cpu=native"`
 - **temp_cache**: Enables temporary caching in functions that retrieve data via the HTTP API.
+- **chrono**: Uses the chrono crate to represent timestamps
+- **time**: Uses the time crate to represent timestamps
```

Comment:
Might be worth mentioning these are mutually exclusive.

```suggestion
- **chrono**\*: Uses the chrono crate to represent timestamps
- **time**\*: Uses the time crate to represent timestamps

....

* -- Only one of these features may be active at a time.
```

Reply:
Wait I just noticed the `time` feature is actually a no-op by now
![Screenshot_20230409_004707](https://user-images.githubusercontent.com/21220820/230745674-327d5430-d081-4378-b0e3-ac4dbae06b56.png)

Will remove the `time` line entirely then